packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -109,9 +110,9 @@ function buildAnswerArrayMetadatum(
     case 'table':
       return buildAnswerArrayForTable(fieldData)
     default:
-      console.log(
-        'This should not happen because check is done in isAnswerArrayValid function',
-      )
+      logger.warn('Unknown fieldtype in answer array', {
+        fieldData,
```

Comment:
Logging all of `fieldData` is dangerous because we risk logging PII like NRICs etc to data dog.

Ideally, I would store these in the logged object:
1. `fieldType `
2. `flowId`
3. `formId`
4. `submissionId`

For now, you can just change `fieldData` to `fieldType`. The rest are not that important to log, and I want this PR landed for good UX improvements.

Reply:
makes sense, I have made some changes, thanks!